mlflow/store/_unity_catalog/registry/rest_store.py

```diff
@@ -207,6 +207,13 @@ def get_model_version_dependencies(model_dir):
                 databricks_dependencies, ResourceType.FUNCTION.value, "DATABRICKS_UC_FUNCTION"
             )
         )
+        dependencies.extend(
```

Comment:
Q: Some resources in [mlflow/models/resources.py](https://github.com/mlflow/mlflow/blob/master/mlflow/models/resources.py#L54) are not listed here, is it intended? If so, can we add comments for that context?

Reply:
This is about unity catalog dependencies. 
genie_space and sql_warehouse are both not UC dependencies. So they don't need to be added here.
I guess the store is Unity_catalog. So I guess that already explains why all resources are not here?

Reply:
Hmm ok maybe I have to learn more about UC. Generally we feel a bit unfamiliar with the dependency related piece of code, so additional references/comments are always appreciated😅 Not a blocker for this PR tho.